src/StatPlots.jl

```diff
@@ -9,11 +9,15 @@ using Distributions
 using DataFrames
 
 import KernelDensity
+import Loess
 @recipe f(k::KernelDensity.UnivariateKDE) = k.x, k.density
 @recipe f(k::KernelDensity.BivariateKDE) = k.x, k.y, k.density
 
 @shorthands cdensity
 
+export groupapply
+export get_summary
```

Comment:
It was proposed by @mkborregaard. The idea is that in the normal scenario, you would have a dataframe that you'd split according to a column and the summarize the split data  (for example getting mean and sem). The use case would be somebody having obtained a grouped dataframe already by some other means and wanting to get the summary, which is not possible with only `groupapply`.
The pipeline is: `groupapply` calls `get_summary` who calls the analysis function on the subdataframes. `groupapply`, `get_summary` and the built-in analysis function have docstrings. The docstrings of `groupapply` refer to `get_summary`, so maybe the user is expecting that it should be exported.